lib/ui/add_book_screen/add_book_screen.dart

```diff
@@ -352,6 +355,17 @@ class _AddBookScreenState extends State<AddBookScreen> {
     });
   }
 
+  void _downloadWork() async {
+    if (widget.work != null) {
+      final OLWork = await OpenLibraryService().getWork(widget.work!);
```

Comment:
please change OLWork to lowerCamelCase like openLibraryWork

Reply:
fixed